src/resolve/resolved-conf.c

```diff
@@ -413,3 +413,47 @@ int manager_parse_config_file(Manager *m) {
 #endif
         return 0;
 }
+
+int config_parse_refuse_record_types(
```

Comment:
Not a matter, but this parser itself is generic, and not specific for `refuse`.
Please rename `config_parse_record_types()`.